app/assets/javascripts/map.js

```diff
@@ -74,26 +74,33 @@ require([
      * you are not completely sure.
      */
     _initViews: function() {
-      // Google Experiments
-      new ExperimentsPresenter();
-
-      var mapView = new MapView();
-
-      new MapControlsView(mapView.map);
-      new TabsView(mapView.map);
-      new AnalysisResultsView();
-      new LayersNavView();
-      new LegendView();
-      new TimelineView();
-      new NavMobileView();
-      new FooterView();
-      new HeaderView();
-      new SourceModalView();
-      new SourceBottomView();
-      new SourceMobileFriendlyView();
-      new NotificationsView();
-      new GuideView();
-      new GuideButtonView();
+
+      var map = new MapView();
+      new MapControlsView(this.map, this.countries);
+
+      this.map = map.map;
+      this.countries = new CountryCollection();
+
+      // Init views
+      this.countries.fetch().done(function(){
```

Comment:
Although the amount of data being downloaded here is small, it does mean that the UI loading is dependent on CartoDB working and returning data in a timely manner. Maybe we should consider having the views below listen to the `sync` event on the collection. For example,

```
this.countries = new CountryCollection();
new TabsView(this.map, this.countries);
... etc ...
this.countries.fetch();
```

and then `TabsView` would have something like `this.listenTo(countries, 'sync', this.render)`. This way parts of the UI that don't rely on the countries can start rendering in the background.

Also, does every view listed here require the countries collection?